fe/tests/stores/skillStore.test.ts

```diff
@@ -20,11 +20,48 @@ describe('Skill Store', () => {
         expect(skillStore.skills).toHaveLength(0)
         expect(skillStore.isLoadingSkills).toBe(false)
 
-        // load skills
-        skillStore.loadSkills()
 
-        // check if it's loading skills
-        expect(skillStore.isLoadingSkills).toBe(true)
+
+        // test.concurrent('concurrent test 1', async () => {
+            // load skills
+            skillStore.loadSkills()
+            // check if it's loading skills
+            expect(skillStore.isLoadingSkills).toBe(true)
+        // })
+
+
+    });
+    it('addSkill', () => {
+
+        // create a new skillstore
+        const skillStore = useSkillStore()
+
+        // check its initial values
+        expect(skillStore.skills).toHaveLength(0)
+        expect(skillStore.isLoadingSkills).toBe(false)
+
+        let callback_finished = false
+
+        test.concurrent('add skill and wait', async () => {
+            // load skills
+            skillStore.addSkill("newSkillName", "color", () => { callback_finished = true})
+
+            expect(callback_finished).toBeTruthy()
+        })
+
+
+    });
+    it('deleteSkill', () => {
+
+        // create a new skillstore
+        const skillStore = useSkillStore()
+
+        // check its initial values
+        expect(skillStore.skills).toHaveLength(0)
+        expect(skillStore.isLoadingSkills).toBe(false)
+
+        // load skills
+        skillStore.deleteSkill(-1)
```

Comment:
Does this really load skills like the comment above says?

Reply:
oops